cves/CVE-2016-3092.yml

```diff
@@ -208,4 +219,8 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
```

Comment:
What do you think could have been done to prevent this mistake?

Reply:
I said that already but I made it more clear on my newest commit